tests/debuginfo/thread-names.rs

```diff
@@ -1,8 +1,8 @@
 //@ compile-flags:-g
-//@ revisions: macos windows
+//@ revisions: macos windows_
```

Comment:
```suggestion
//@ revisions: macos win
```
(For consistency with other tests.)

Reply:
Also, maybe `#![allow]` would work better for cases like this?

Reply:
> Also, maybe `#![allow]` would work better for cases like this?

I have a slight preference to not disturb test with unnecessary things, like allow of lints, strait lines, ... and be as straight as to the point.